cypress/integration/userCanChooseFood.feature.js

```diff
@@ -13,11 +13,12 @@ describe("Restaurants", () => {
     cy.chooseHotel()
     cy.get(".fluid > .dropdown").click();
     cy.get(".active > .visible > :nth-child(3)").click();
+    
 
     cy.get("#food-slider[type=range]").then(input =>
       changeRangeInputValue(input)(3)
     );
-    cy.get(":nth-child(2) > .food-choice > .ui").click()
+    cy.get(":nth-child(3) > .food-choice > .ui").click()
```

Comment:
I gave that button an ID earlier so change to 
`cy.get("#find-restaurants").click()`
instead of having to mess around with this crap

Reply:
thx